test/routes/viewModels/taskListModelsTest.js

```diff
@@ -2145,68 +2190,13 @@ describe('TaskList models', () => {
           label: 'Not completed',
           title: 'Proposed curfew address',
         },
-        {
-          action: {
-            href: '/hdc/review/victimLiaison/',
-            text: 'View',
-            type: 'btn-secondary',
-          },
-          label: 'Not completed',
-          title: 'Victim liaison',
-        },
-        {
-          action: {
-            href: '/hdc/review/curfewHours/',
-            text: 'View',
-            type: 'btn-secondary',
-          },
-          label: 'Not completed',
-          title: 'Curfew hours',
-        },
-        {
-          action: {
-            href: '/hdc/review/conditions/',
-            text: 'View',
-            type: 'btn-secondary',
-          },
-          label: 'Not completed',
-          title: 'Additional conditions',
-        },
-        {
-          action: {
-            href: '/hdc/review/reporting/',
-            text: 'View',
-            type: 'btn-secondary',
-          },
-          label: 'Not completed',
-          title: 'Reporting instructions',
-        },
-        {
-          action: {
-            href: '/hdc/review/finalChecks/',
-            text: 'View',
-            type: 'btn-secondary',
-          },
-          label: 'Not completed',
-          title: 'Review case',
-        },
-        {
-          action: {
-            href: '/hdc/send/return/',
-            text: 'Return to prison case admin',
-            type: 'btn-secondary',
-          },
-          title: 'Return to prison case admin',
-        },
-        {
-          action: {
-            href: '/hdc/approval/release/',
-            text: 'Continue',
-            type: 'btn',
-          },
-          label: 'Not started',
-          title: 'Final decision',
-        },
+        victim,
+        curfew,
+        conditions,
+        reporting,
+        review,
+        returnPCA,
+        release,
```

Comment:
just removing some of the endless duplication in these tests. More removals to do later, but want to keep this change small